<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FindPwd</class>
 <widget class="QMainWindow" name="FindPwd">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>573</width>
    <height>513</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="mail_label">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>70</y>
      <width>63</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>安全邮箱</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="mail_edit">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>60</y>
      <width>221</width>
      <height>36</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="mail_hint">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>70</y>
      <width>63</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="pwd_label">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>140</y>
      <width>63</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>密码</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="pwd_edit">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>130</y>
      <width>231</width>
      <height>36</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="pwd_confirm_label">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>190</y>
      <width>63</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>确认密码</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="pwd_confirm_edit">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>190</y>
      <width>231</width>
      <height>36</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="pwd_confirm_hint">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>190</y>
      <width>63</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="reset_btn">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>310</y>
      <width>161</width>
      <height>36</height>
     </rect>
    </property>
    <property name="text">
     <string>重置</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>573</width>
     <height>36</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
